.github/workflows/build.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
```

Comment:
You might get better performance if you upgrade the action https://github.com/JupiterOne/.github/blob/main/.github/workflows/npm_package_pr_npm.yaml#L40. Not blocking

Reply:
Let me make that change in a future PR so I can wrap this up. I already versioned this branch and don't want to deal with fixing that :)